tests/support/fixtures.ts

```diff
@@ -235,6 +257,7 @@ export async function createSeedFixture() {
   await bob.connect(palm);
 
   await alice.git(["clone", sourceBrowsingDir], { cwd: alice.checkoutPath });
+  alice.setCwd(Path.join(alice.checkoutPath, "source-browsing"));
```

Comment:
I would rather prefer if we pass in the path to each command that we run explicitly, so that there is only one way do do it and it is explicit. I feel like having this extra state in the peer could make it easier to make mistakes.

Reply:
ok, I found it verbose to add `{ cwd: Path.join(alice.checkoutPath, "source-browsing" }` to a lot of commands.
But maybe I can save it to a `projectCheckoutPath` variable and reference it like this

Reply:
8abe9a6